src/applications/toe/config/form.js

```diff
@@ -807,48 +807,7 @@ const formConfig = {
                 'ui:description': LearnMoreAboutMilitaryBaseTooltip(),
               },
               [formFields.address]: {
-                ...address.uiSchema('', false, null, true),
-                country: {
-                  'ui:title': 'Country',
-                  'ui:required': formData =>
-                    !formData['view:mailingAddress'].livesOnMilitaryBase,
-                  'ui:disabled': formData =>
-                    formData['view:mailingAddress'].livesOnMilitaryBase,
-                  'ui:options': {
-                    updateSchema: (formData, schema, uiSchema) => {
-                      const countryUI = uiSchema;
-                      const addressFormData = get(
-                        ['view:mailingAddress', 'address'],
-                        formData,
-                      );
-                      const livesOnMilitaryBase = get(
-                        ['view:mailingAddress', 'livesOnMilitaryBase'],
-                        formData,
-                      );
-                      if (livesOnMilitaryBase) {
-                        countryUI['ui:disabled'] = true;
-                        const USA = {
-                          value: 'USA',
-                          label: 'United States',
-                        };
-                        addressFormData.country = USA.value;
-                        return {
-                          enum: [USA.value],
-                          enumNames: [USA.label],
-                          default: USA.value,
-                        };
-                      }
-                      countryUI['ui:disabled'] = false;
-                      return {
-                        type: 'string',
-                        enum: constants.countries.map(country => country.value),
-                        enumNames: constants.countries.map(
-                          country => country.label,
-                        ),
-                      };
-                    },
-                  },
-                },
+                ...address.uiSchema(''),
```

Comment:
Think you must have had a bad merge. This is wiping out all of the military address changes I worked on recently.